services/QuillLMS/app/models/response.rb

```diff
@@ -0,0 +1,145 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: responses
+#
+#  id                            :bigint           not null, primary key
+#  answer                        :jsonb
+#  attempt_number                :integer
+#  correct                       :boolean          not null
+#  extra_metadata                :jsonb
+#  question_number               :integer
+#  question_score                :float
+#  created_at                    :datetime         not null
+#  activity_session_id           :integer          not null
+#  concept_id                    :integer
+#  concept_result_id             :integer
+#  response_directions_id        :integer
+#  response_instructions_id      :integer
+#  response_previous_feedback_id :integer
+#  response_prompt_id            :integer
+#  response_question_type_id     :integer
+#
+# Indexes
+#
+#  index_responses_on_concept_result_id  (concept_result_id) UNIQUE
+#
+class Response < ApplicationRecord
```

Comment:
Naming-wise, does response still make sense given how the scope has changed? Should this be something like `concept_results`?